### DISCUSSION TITLE
Error loading data

### DISCUSSION THREAD

**Discussion by chrisb2023** [KelvinTegelaar/CIPP#1483] [Q&A]
Hi everyone,

I'm after some help.  We have enrolled in the SAM Setup wizard (via CIPP creating a new SAM application for me).  All of the tenants have pulled through but upon selecting any Tenant and clicking on any of the options we are getting "Error loading data".
Running a Tenant Access Check we get the following two errors:

(1). Failed to connect to invalid_grant:AADSTS90072: User account '{EmailHidden}'
(2). Failed to connect to Exchange: Failed to obtain Classic API Token 

The errors are suggesting it's a permission issue.  I have verified within all the tenants which we have a partner relationship under DAP and all relationships contains the role Global Administrator.  Do i need to do anything within each clients Tenant to allow access/permission?  I'm sure this is going to be a simple resolution . . . . can someone point me in the right direction?  Thanks

**Comment by KelvinTegelaar** [comment]
This is not the place for support, as a sponsor you get basic email support, non sponsors can use the discord community to or use our docs at cipp.app.
